### PR TITLE
Add `curated-transformers.PyTorchCheckpointLoader.v1`

### PR DESCRIPTION
**Pull request by danieldk**
This loader deserializes a model (with HF parameter naming) from a local PyTorch checkpoint file.

### CODE REVIEW

File: curated_transformers/models/hf_util.py [hunk 1]
```diff
@@ -33,15 +49,25 @@ def convert_hf_pretrained_model_parameters(
...
 
 
 def _convert_albert_base_state(
-    hf_model: "transformers.PreTrainedModel",
+    params: OrderedDict[str, torch.Tensor],
 ) -> Dict[str, torch.Tensor]:
-    out = {}
-
-    state_dict = dict(hf_model.state_dict().items())
+    # Strip the `bert` prefix from XLM-Roberta model parameters.
```
Comment (shadeMe) [inline curated_transformers/models/hf_util.py:69]: This comment seems incorrect? `bert` should be `albert` according to the substitution below, and I think you mean "`Albert` model parameters"?

File: curated_transformers/models/hf_util.py [hunk 2]
```diff
@@ -83,33 +109,34 @@ def _convert_albert_base_state(
...
 
 
 def _convert_bert_base_state(
-    hf_model: "transformers.PreTrainedModel",
+    params: OrderedDict[str, torch.Tensor],
 ) -> Dict[str, torch.Tensor]:
     out = {}
 
-    state_dict = dict(hf_model.state_dict().items())
+    # Strip the `bert` prefix from XLM-Roberta model parameters.
```
Comment (shadeMe) [inline curated_transformers/models/hf_util.py:136]:
```suggestion
    # Strip the `bert` prefix from BERT model parameters.
```

File: curated_transformers/models/hf_util.py (hunk 1 above)
Comment (danieldk) [reply curated_transformers/models/hf_util.py]: Fixed both comments.

**Comment by danieldk** [comment]
Fixed the merge conflict.

**Review by shadeMe — APPROVED** [review]

**Pull request merged**